src/Core/Core.csproj

```diff
@@ -34,9 +34,9 @@
     <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="3.0.0" />
-    <PackageReference Include="Microsoft.Quantum.Compiler" Version="0.12.20062505-beta" />
-    <PackageReference Include="Microsoft.Quantum.CsharpGeneration" Version="0.12.20062505-beta" />
-    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.12.20062505-beta" />
+    <PackageReference Include="Microsoft.Quantum.Compiler" Version="0.11.2006.2203-beta" />
+    <PackageReference Include="Microsoft.Quantum.CsharpGeneration" Version="0.11.2006.2203-beta" />
+    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.11.2006.2203-beta" />
```

Comment:
I'd recommend reverting this change before merging so that we keep the newer version, since I think @cgranade's recent commit depends on having a new-enough build here.